Explorer/Assets/DCL/Multiplayer/Movement/Systems/SDKAvatarShapesMotionSystem.cs

```diff
@@ -0,0 +1,108 @@
+using Arch.Core;
+using Arch.System;
+using Arch.SystemGroups;
+using Arch.SystemGroups.DefaultSystemGroups;
+using DCL.AvatarRendering.AvatarShape.UnityInterface;
+using DCL.Character.Components;
+using DCL.CharacterMotion.Animation;
+using DCL.CharacterMotion.Components;
+using DCL.Diagnostics;
+using DCL.Profiles;
+using ECS.Abstract;
+using UnityEngine;
+
+namespace DCL.Multiplayer.Movement.Systems
+{
+    [UpdateInGroup(typeof(PresentationSystemGroup))]
+    [LogCategory(ReportCategory.AVATAR)]
+    public partial class SDKAvatarShapesMotionSystem : BaseUnityLoopSystem
+    {
+        private const float ROTATION_SPEED = 5f;
+        private const float RUN_SPEED_THRESHOLD = 9.5f;
+        private const float JOG_SPEED_THRESHOLD = 4f;
+
+        private SDKAvatarShapesMotionSystem(World world) : base(world) { }
+
+        protected override void Update(float t)
+        {
+            UpdateMotionQuery(World);
+        }
+
+        [Query]
+        [None(typeof(Profile))]
+        private void UpdateMotion(
+            in IAvatarView view,
+            ref CharacterTransform characterTransformComponent,
+            ref CharacterLastPosition characterLastPositionComponent,
+            ref CharacterAnimationComponent animationComponent)
+        {
+            UpdateRotation(characterTransformComponent, ref characterLastPositionComponent);
+            UpdateAnimations(view, characterTransformComponent, ref characterLastPositionComponent, ref animationComponent);
+            UpdateLastPosition(ref characterTransformComponent, ref characterLastPositionComponent);
```

Comment:
`characterTransformComponent `is times without `ref `but last one with. Is there any reason? if not, then consistency is better

Reply:
Done: https://github.com/decentraland/unity-explorer/pull/3297/commits/7177ba02625d0fd5207f7428cd6177c7e53d0e40